src/main/java/mat/server/util/MeasureUtility.java

```diff
@@ -40,8 +40,19 @@ public static String getVersionText(String version, boolean isDraft) {
     public static String formatVersionText(String version) {
         StringUtility su = new StringUtility();
         String[] versionArr = version.split("\\.");
-        String majorVersion = su.trimLeadingZeros(versionArr[0]);
-        String minorVersion = su.trimLeadingZeros(versionArr[1]);
+        String majorVersion = "";
+        String minorVersion = "";
+        String revisionNumber = "";
+        for(int i = 0; i < versionArr.length; i++) {
+            if(i == 0) {
+                majorVersion = su.trimLeadingZeros(versionArr[0]);
+            } else if(i == 1) {
+                minorVersion = su.trimLeadingZeros(versionArr[1]);
+            } else {
+                revisionNumber = su.trimLeadingZeros(versionArr[2]);
+                return majorVersion + "." + minorVersion + "." + revisionFormat.format(Integer.parseInt(revisionNumber));

```

Comment:
For revisionFormat, is there a benefit to converting the String to an int then formatting back to a String?

Reply:
Revision Number is the one that is causing an issue in MAT dev, not exactly sure why though. This is just a fix but was unable to find the root cause.

Reply:
Nevermind, I see now that it is converting the string to an int to use DecimalFormat to add the left padding.